public/mocks/v1_get_place_node_attributes.json

```diff
@@ -0,0 +1,767 @@
+        "includedColumns": [
+          {
+            "name": "Soy deforestation (until 2014)",
+            "unit": "ha"
+          },
+          {
+            "name": "Number of environmental embargoes (2015)"
+          },
+          {
+            "name": "Area affected by fires (2013)",
+            "unit": "km2"
+          },
+          {
+            "name": "Cases of forced labor (2014)"
+          },
+          {
+            "name": "Cases of land conflicts (2014)"
+          },
+          {
+            "name": "Biodiversity (loss of amphibian habitat)"
+          },
+          {
+            "name": "Annual deforestation rate (2015)",
+            "unit": "ha"
+          },
+          {
+            "name": "Human development index (2013)"
+          },
+          {
+            "name": "Criticality of water scarcity (2013)"
+          }
+        ],
+        "rows": [
+          {
+            "name": "Score",
+            "values": [
+              null,
+              2.0,
+              29.55,
+              0.0,
+              0.0,
+              null,
+              1840.0,
+              0.69,
+              2.0
+            ]
+          },
+          {
+            "name": "State Ranking",
+            "values": [
+              13,
+              27,
+              4,
+              51,
+              70,
+              6,
+              30
+            ]
+          }
+        ]
+      },
+      {
+        "name": "Socio-economic indicators",
+        "includedColumns": [
+          {
+            "name": "Soy production (2015)",
+            "unit": "Tn"
+          },
+          {
+            "name": "Cattle",
+            "unit": "heads"
+          },
+          {
+            "name": "Soy yield",
+            "unit": "Tn/ha"
+          },
+          {
+            "name": "Area soy",
+            "unit": "%"
+          },
+          {
+            "name": "Area maize",
+            "unit": "%"
+          },
+          {
+            "name": "Area cotton",
+            "unit": "%"
+          }
+        ],
+        "rows": [
+          {
+            "name": "Score",
+            "values": [
+              null,
+              2.0,
+              29.55,
+              0.0,
+              0.0,
+              null,
+              1840.0,
+              0.69,
+              2.0
+            ]
+          },
+          {
+            "name": "State Ranking",
+            "values": [
+              13,
+              27,
+              4,
+              51,
+              70,
+              6,
+              30
+            ]
+          }
+        ]
+      }
+    ],
+    "trajectory_deforestation": {
+      "includedYears": [
+        2003,
+        2004,
+        2005,
+        2006,
+        2007,
+        2008,
+        2009,
+        2010,
+        2011,
+        2012,
+        2013,
+        2014,
+        2015
+      ],
+      "lines": [
+        {
+          "name": "Soy related deforestation",
+          "type": "area",
```

Comment:
Same answer as above, we consider that the graph type (line or area chart) is too tied to the meaning of it to have this on the front-end, but it's debatable.